CoreDataAccessView/test/unit/src/au/gov/asd/tac/constellation/views/dataaccess/state/DataAccessPreferenceKeysNGTest.java

```diff
@@ -0,0 +1,676 @@
+/*
+ * Copyright 2010-2021 Australian Signals Directorate
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.views.dataaccess.state;
+
+import static au.gov.asd.tac.constellation.views.dataaccess.state.DataAccessPreferenceKeys.DESELECT_PLUGINS_ON_EXECUTE_PREF;
+import static au.gov.asd.tac.constellation.views.dataaccess.state.DataAccessPreferenceKeys.PREVIOUS_DATA_DIR_PREF;
+import static au.gov.asd.tac.constellation.views.dataaccess.state.DataAccessPreferenceKeys.SAVE_DATA_DIR_PREF;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.prefs.Preferences;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.validateMockitoUsage;
+import static org.mockito.Mockito.when;
+import org.mockito.MockitoAnnotations;
+import org.openide.util.Exceptions;
+import org.openide.util.NbPreferences;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.fail;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+/**
+ *
+ * @author aldebaran30701
+ */
+public class DataAccessPreferenceKeysNGTest {
+
+    Preferences prefs;
+
+    public DataAccessPreferenceKeysNGTest() {
+    }
+
+    @BeforeClass
+    public void setUpClass() throws Exception {
+    }
+
+    @AfterClass
+    public static void tearDownClass() throws Exception {
+    }
+
+    @BeforeMethod
+    public void setUpMethod() throws Exception {
+        // Before each method, open the mocks.
+        MockitoAnnotations.openMocks(this);
```

Comment:
I think this only needs to be called if you are using Mock annotations which it doesn't appear so. Correct me if I'm wrong